config/packages/netgen_notifications.yaml

```diff
@@ -0,0 +1,23 @@
+netgen_notifications:
+    categories:
+        default:
+            subscription:
+                enabled: true
+                resolver: netgen_notifications.ibexa.subtree_update_resolver
+                digest_actions:
+                    email:
+                        type: digest_email
+                        parameters:
+                            from: 'info@netgen.io'
+                            subject: 'Here is what you missed'
```

Comment:
I am not sure of the scope of work here, but the subject should probably be translateable?

Reply:
Not possible currently. Limitation of the notifications bundle.